frontend/test/metabase/scenarios/dashboard-filters/dashboard-filters-source.cy.spec.js

```diff
@@ -192,7 +192,6 @@ const setupNativeQuestionSource = () => {
 const setupCustomList = () => {
   modal().within(() => {
     cy.findByText("Custom list").click();
-    cy.findByRole("textbox").should("contain.value", "Gizmo");
```

Comment:
Why is this assertion removed?

Reply:
Because we no longer prefill static list values with field values